docs/UserGuide.md

```diff
@@ -50,16 +50,18 @@ Avengers Assemble (AA) is a **desktop app for managing contacts, optimized for u
 
 * Some commands require you to include parameters. These parameters are identified by prefixes.
 
-* Here are a list of valid prefixes and what they each refer to.
-
-| Prefix | What it refers to         |
-|--------|---------------------------|
-|n/      | Name of the person        |
-|p/      | Phone number of person    |
-|e/      | Email of person           |
-|a/      | Address of person         |
-|t/      | Tags of person            |
-|m/      | Matriculation ID of person|
+* Here are a list of valid prefixes and what they each refer to. 
+* Prefixes encased with '[ ]' are optional.
+* Prefixes with '…' after them can be used multiple times.
+
+| Prefix  | What it refers to          | Constraints                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
```

Comment:
good job adding constraints into the table as I think it improves the readability of the user guide and allows users to follow the user guide easier